src/components/CodeEditor.tsx

```diff
@@ -263,7 +267,7 @@ export const CodeEditor = ({
                 for (let pos = from; pos < to; ) {
                   const line = view.state.doc.lineAt(pos)
                   const lineText = line.text
-                  const matches = lineText.matchAll(/@tsci\/[\w\-.]+/g)
+                  const matches = lineText.matchAll(/@tsci\/[\w.]+/g)
```

Comment:
this would introduce a bug, revert

Reply:
I am sorry for all this trouble i am thinking to shut this pr and create a cleaner pr that is easier to manage for you and ofcourse myself

Reply:
yes that's always ok!

Reply:
and no problem- my messages are short because i often type them from my phone, not because i'm angry 😁